src/views/ManageList.jsx

```diff
@@ -31,10 +30,39 @@ export function ManageList({ userId }) {
 		const listPath = localStorage.getItem('tcl-shopping-list-path');
 
 		if (!listPath) {
-			window.alert('List is not existed.');
+			window.alert('List does not exist.');
 			return;
 		}
 
+		const formDataCheck = formData.name
+			.replace(/[^A-Z0-9]/gi, '')
+			.toLowerCase();
+
+		if (
+			data.find(
+				(item) =>
+					item.name.trim().toLowerCase() === formData.name.trim().toLowerCase(),
+			)
+		) {
```

Comment:
<!-- Ask the author for clarification or investigation.  -->
**question (non-blocking):** The number of newlines here  and on lines 54-53 below make it kind of hard to read and are more Java-esque than JavaScript-esque - were those added by the linter?

Reply:
I believe so! checking that out now

Reply:
Changed it, though It looks like the linter adds it back after my commit.